charts/kpow/templates/ingress.yaml

```diff
@@ -1,9 +1,9 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "kpow.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
`.Capabilities.KubeVersion.GitVersion` was deprecated in favor of `.Capabilities.KubeVersion.Version`
https://helm.sh/docs/chart_template_guide/builtin_objects/